drivers/media/mc/mc-request.c

```diff
@@ -414,8 +414,8 @@ int media_request_object_bind(struct media_request *req,
 
 	spin_lock_irqsave(&req->lock, flags);
 
-	if (WARN_ON(req->state != MEDIA_REQUEST_STATE_UPDATING))
-		goto unlock;
+//	if (WARN_ON(req->state != MEDIA_REQUEST_STATE_UPDATING))
```

Comment:
Remove rather than comment out.

Reply:
Don't actually need this change if using the pin code so I've removed the change